src/lammpsio/topology.py

```diff
@@ -0,0 +1,121 @@
+import numpy
+
+
+class Topology:
+    def __init__(self, N, num_members):
+        self._N = int(N)
```

Comment:
Document that `N` is an int, and make it the caller's responsibility to cast
```suggestion
        self._N = N
```